### PR TITLE
Add nextdenovo

### PR DESCRIPTION
**Pull request by loraine-gueguen**
Add nextdenovo.

One test was successful. 
The second test (with test data), was only partially successful: some jobs have been submitted, but others are killed before completion. I don't know why (https://dev.azure.com/bioconda/bioconda-recipes/_build/results?buildId=23822&view=logs&j=e14e69ff-a0ae-55c4-b71d-229b239cfb2f&t=7df82132-b284-504b-53d6-7d3e63519572&l=868)

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by loraine-gueguen** [comment]
@BiocondaBot please add label

**Comment by dpryan79** [comment]
@BiocondaBot please fetch artifacts

File: recipes/nextdenovo/build.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -x
+
+SHARE_DIR=${PREFIX}/share/${PKG_NAME}-$PKG_VERSION-$PKG_BUILDNUM
+mkdir -p ${PREFIX}/bin ${SHARE_DIR}
+cp -r ./* ${SHARE_DIR}
+chmod a+x ${SHARE_DIR}/nextDenovo
+ln -s ${SHARE_DIR}/nextDenovo ${PREFIX}/bin/nextDenovo
+export PYTHONPATH=${PYTHONPATH}:${SHARE_DIR}/lib
```
Comment (dpryan79) [inline recipes/nextdenovo/build.sh:9]: This is only run once when the package is built and not when users install the package.

**Comment by dpryan79** [comment]
Is this repackaging binaries? Then we need to restrict it to Linux and make sure the dependencies (xz, etc.) exist. Do we need to repackage those or can we instead build them as part of the recipe? They'll be more likely to work if that's the case.

**Comment by loraine-gueguen** [comment]
> Is this repackaging binaries?

Binaries are provided as is in the archive. We do not have all source files.

> Then we need to restrict it to Linux and make sure the dependencies (xz, etc.) exist. Do we need to repackage those or can we instead build them as part of the recipe? They'll be more likely to work if that's the case.

Do you mean something like ?

```
requirements:
  host:
    - python
    - bzip2
    - zlib
    - xz
    - and so on....
```

File: recipes/nextdenovo/build.sh (hunk 1 above)
Comment (loraine-gueguen) [reply recipes/nextdenovo/build.sh:9]: ok. Should it be ok with someting like (for the python modules) ?

`ln -s ${SHARE_DIR}/lib/* ${PREFIX}/lib/.`

Comment (loraine-gueguen) [reply recipes/nextdenovo/build.sh]: fixed in https://github.com/bioconda/bioconda-recipes/pull/37859/commits/d1e9f33665f83616e483c352e06237e87cb5747f

**Comment by loraine-gueguen** [comment]
I could not set `skip: True # [osx]` because this is not compatible with `noarch`, asked by lint (as there is no compiler)...

**Comment by dpryan79** [comment]
@BiocondaBot please fetch artifacts

**Comment by loraine-gueguen** [comment]
thanks for you help @dpryan79 !

**Comment by loraine-gueguen** [comment]
Is it ready to be merged ?

**Comment by dpryan79** [comment]
@BiocondaBot please fetch artifacts

**Review by dpryan79 — APPROVED** [review]

**Comment by dpryan79** [comment]
@BiocondaBot please merge

@loraine-gueguen Yes! Sorry, this fell off my radar.

**Pull request merged**